docs/sources/flow/concepts/modules.md

```diff
@@ -145,3 +251,4 @@ loki.write "default" {
 [Components]: "/docs/agent/ -> /docs/agent/<AGENT_VERSION>/flow/reference/components"
 [Components]: "/docs/grafana-cloud/ -> /docs/grafana-cloud/send-data/agent/flow/reference/components"
 {{% /docs/reference %}}
+{{< /collapse >}}
```

Comment:
We can't nest `docs/reference` blocks inside another shortcode (known issue). The links in the block need to be stripped out of the `docs/reference` block, reformatted to a regular rel-ref definition, and placed just under `[Module loader]: #module-loaders`.  In this case we don't need to be too concerned with the Grafana Cloud links.

See: https://grafana.com/docs/writers-toolkit/write/links/#link-to-grafanacom-pages

```
[argument block]: https://grafana.com/docs/agent/<AGENT_VERSION>/flow/reference/config-blocks/argument
[export block]: https://grafana.com/docs/agent/<AGENT_VERSION>/flow/reference/config-blocks/export
[Component controller]: https://grafana.com/docs/agent/<AGENT_VERSION>/flow/concepts/component_controller
[Components]: https://grafana.com/docs/agent/<AGENT_VERSION>/flow/reference/components
```